.github/workflows/release-dependencies-diff-generator.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -el
+# Forked from https://github.com/usefulness/dependency-tree-diff-action/blob/master/entrypoint.sh
+# This version allows running a different dependency task which is necessary when mixing java/android modules.
+
+wget "https://github.com/JakeWharton/dependency-tree-diff/releases/download/$INPUT_VERSION/dependency-tree-diff.jar" -q -O dependency-tree-diff.jar
+
+cd "$INPUT_BUILD_ROOT_DIR"
+
+baselineDependenciesDir=$INPUT_BUILD_ROOT_DIR/build/release-dependencies-diff/baseline-dependencies
+dependenciesDir=$INPUT_BUILD_ROOT_DIR/build/release-dependencies-diff/dependencies
```

Comment:
You can provide only `baseline-dependencies`/`dependencies` part as a parameter and build the full path in Gradle via `project.rootProject.buildForlder` or something like this.

Reply:
yeah, I think I'll go with that. it seems to be having problems with obtaining the file path from the shell script =|